examples/set-labels-full-coverage/README.md

```diff
@@ -58,4 +58,4 @@ $ kpt fn render set-labels-full-coverage
 Check all resources, the following labels should be upserted to the `labels`, `matchingLabels` or `seletors(labelSelector)` fields: 
 `color: orange`, `fruit: apple` and `app: new`.
 
-[`set-labels`]: https://catalog.kpt.dev/set-labels/v0.1/
\ No newline at end of file
+[`set-labels`]: https://catalog.kpt.dev/set-labels/v0.2/
```

Comment:
This website (https://catalog.kpt.dev/set-labels/v0.2/) isn't available yet, is that expected?

Reply:
Yes, this PR is to update the website